Tanish/armstrongnumber.cpp

```diff
@@ -0,0 +1,19 @@
+#include<iostream>
+#include<math.h>
+using namespace std;
+
+int main()
+{
+    int n,dig=0,sum=0;
+    cin>>n;
+    for(int i=n;i>0;i/=10)
+        dig++;
+    for(int i=n;i>0;i/=10)
+        sum+=pow(i%10,dig);
+    cout<<sum<<endl;
+    if(sum==n)
```

Comment:
Add a line before conditional statements

Reply:
resolved this issue